reference/5.0/Microsoft.PowerShell.Management/New-PSDrive.md

```diff
@@ -33,26 +33,22 @@ However, because temporary drives are known only to PowerShell, you cannot acces
 
 The following features are added to `New-PSDrive` in Windows PowerShell 3.0:
 
-* **Mapped network drives**
-
-  You can use the **Persist** parameter of `New-PSDrive` to create Windows mapped network drives.
+- Mapped network drives.
+  You can use the *Persist* parameter of `New-PSDrive` to create Windows mapped network drives.
   Unlike temporary PowerShell drives, Windows mapped network drives are not session-specific.
   They are saved in Windows and they can be managed by using standard Windows tools, such as File Explorer and Net Use.
   Mapped network drives must have a drive-letter name and be connected to a remote file system location.
-  When your command is scoped locally (no dot-sourcing), the **Persist** parameter does not persist the creation of a **PSDrive** beyond the scope in which the command is running.
+  When your command is scoped locally (no dot-sourcing), the *Persist* parameter does not persist the creation of a **PSDrive** beyond the scope in which the command is running.
   If you are running `New-PSDrive` inside a script, and you want the drive to persist indefinitely, you must dot-source the script.
-  For best results, to force a new drive to persist indefinitely, add the **Scope** parameter to your command, and set its value to Global.
-
-* **External drives**
-
+  For best results, to force a new drive to persist indefinitely, add the *Scope* parameter to your command, and set its value to Global.
+- External drives.
   When an external drive is connected to the computer, PowerShell automatically adds a **PSDrive** to the file system that represents the new drive.
   You do not have to restart PowerShell.
   Similarly, when an external drive is disconnected from the computer, PowerShell automatically deletes the **PSDrive** that represents the removed drive.
+- Credentials for UNC Paths.
 
-* **Credentials for UNC Paths**
-
-  When the value of the **Root** parameter is a UNC path, such as \\\\Server\Share, the credential specified in the value of the **Credential** parameter is used to create the **PSDrive**.
-  Otherwise, **Credential** is not effective when you are creating new file system drives.
+When the value of the *Root* parameter is a UNC path, such as \\\\Server\Share, the credential specified in the value of the *Credential* parameter is used to create the **PSDrive**.

```

Comment:
- Property, parameter, and class names should be **bold**

Reply:
Fixed